src/Sanity.Linq/JsonConverters/SanityReferenceTypeConverter.cs

```diff
@@ -66,6 +66,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
                 //Get reference from object
                 var valRef = type.GetProperty("Ref").GetValue(value) as string;
+                object Value = null;
```

Comment:
Bit picky here, but we should use camelCase instead of PascalCase for local variable names.